<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>825</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Administrador de Procesos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>784</width>
      <height>621</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="title">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="orden_1">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="orden_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pausar_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pausar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="T_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>430</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>352</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>460</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>570</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar_1">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pausar_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pausar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="T_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pausar_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pausar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>350</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>570</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>460</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>570</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="T_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pausar_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pausar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pausar_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Pausar</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>350</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>460</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="orden_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>430</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelar_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>240</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="orden_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>540</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="T_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="orden_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>210</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T. CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>520</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T. CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>410</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T. CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>300</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T. CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>190</y>
       <width>47</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>T. CPU</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>520</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>410</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>300</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>190</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <zorder>progressBar_1</zorder>
    <zorder>pausar_5</zorder>
    <zorder>cancelar_3</zorder>
    <zorder>cancelar_4</zorder>
    <zorder>iniciar_5</zorder>
    <zorder>cancelar_1</zorder>
    <zorder>pausar_1</zorder>
    <zorder>iniciar_2</zorder>
    <zorder>iniciar_1</zorder>
    <zorder>pausar_4</zorder>
    <zorder>iniciar_3</zorder>
    <zorder>cancelar_5</zorder>
    <zorder>progressBar_4</zorder>
    <zorder>progressBar_5</zorder>
    <zorder>T_1</zorder>
    <zorder>pausar_2</zorder>
    <zorder>pausar_3</zorder>
    <zorder>progressBar_3</zorder>
    <zorder>progressBar_2</zorder>
    <zorder>iniciar_4</zorder>
    <zorder>orden_4</zorder>
    <zorder>cancelar_2</zorder>
    <zorder>orden_5</zorder>
    <zorder>T_5</zorder>
    <zorder>orden_2</zorder>
    <zorder>T_2</zorder>
    <zorder>orden_1</zorder>
    <zorder>orden_3</zorder>
    <zorder>T_4</zorder>
    <zorder>T_3</zorder>
    <zorder>reset_btn</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
   </widget>
   <widget class="QPushButton" name="lotes_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lotes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multi_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>341</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiprogramación</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="quantum">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Quantum</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="rr_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Round Robin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fcfs_btn">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>FCFS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
